scripts/check_if_pr_is_low_risk_test.py

```diff
@@ -183,14 +186,14 @@ def mock_run_cmd(tokens):
         self.assertListEqual(diff_files, expected_diff_files)
         self.assertDictEqual(file_diffs, expected_file_diffs)
 
-    def test_name_status_failure(self):
+    def test_name_status_failure(self) -> None:
 
-        def mock_run_cmd(unused_tokens):
+        def mock_run_cmd(unused_tokens: List[str]) -> str:
```

Comment:
I think we should not do that because if change all unused args with `_` then it creates ambiguity among other developers about what variable denote what. Taking an example:

example is taken from https://github.com/oppia/oppia/blob/953dc2b1d39f604bca41db8540b865140e14a808/scripts/run_mypy_checks_test.py#L55
```python
        def mock_popen_success(
            unused_cmd: str,
            stdout: Optional[str] = None,  # pylint: disable=unused-argument
            stdin: Optional[str] = None,  # pylint: disable=unused-argument
            stderr: Optional[str] = None,  # pylint: disable=unused-argument
            env: Optional[str] = None  # pylint: disable=unused-argument
        ) -> subprocess.Popen[bytes]:  # pylint: disable=unsubscriptable-object
            return process_success
```
```python
        def mock_popen_success(
            _: str,
            _: Optional[str] = None,
            _: Optional[str] = None, 
            _: Optional[str] = None, 
            _: Optional[str] = None 
        ) -> subprocess.Popen[bytes]:
            return process_success
```

I think the first one looks better than the second one, I need your opinion!

Reply:
You could also use `unused_` as prefix just like `unused_` to silence the pylint error.